makuai-corp-site/static/css/contact.css

```diff
@@ -0,0 +1,203 @@
+.header__nav-link--contact::after {
+    opacity: 1;
+}
+.contact {
+    max-width: 880px;
+    padding: 188px 0 100px;
+    margin: 0 auto;
+}
+.contact__heading {
+    position: relative;
+    width: fit-content;
+    margin: 0 auto 24px;
+}
+.contact__heading::after {
+    position: absolute;
+    top: 1em;
+    left: 0px;
+    right: 0px;
+    z-index: -1;
+    height: 0.4em;
+    border-radius: 3px;
+    content: "";
+    background: var(--main-heading-decoration);
+    opacity: 0.2;
+}
+.contact__text {
+    padding: 0 16px;
+    margin: 0 auto 28px;
+    text-align: center;
+}
+.contact__form {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    gap: 4px;
+    max-width: 560px;
+    padding: 0 16px;
+    margin: auto;
+}
+.contact__form-label {
+    font-family: 'Noto Sans JP', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 180%;
+    letter-spacing: 0.02em;
+    color: #333333;
+    margin-top: 20px;
+}
+.contact__form-input {
+    display: block;
+    width: 100%;
+    padding: 6px 20px;
+    font-family: 'Noto Sans JP', sans-serif;
+    font-size: 22px;
+    color: #333333;
+    font-weight: 300;
+    line-height: 200%;
+    letter-spacing: 0.02em;
+    border-radius: 8px;
+    border: 1px solid #FFFFFF;
+    box-sizing: border-box;
+    outline: 1px solid #333333;
+}
+.contact__form-input:focus {
+    outline: none;
+    width: calc(100% - 2px);
+    height: calc(100% - 2px);
+    padding-left: 19px;
+    margin: 1px;
+}
+.contact__form-input-outline {
+    position: relative;
+    max-width: 560px;
+}
+.contact__form-input-outline::after {
+    position: absolute;
+    z-index: -1;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    content: "";
+    border-radius: 8px;
+    background: linear-gradient(90deg, #45D1D1 0%, #ED5D5D 51.56%, #F8BF2D 100%);
```

Comment:
```suggestion
    position: absolute;
    z-index: -1;
    /* top: 0; left: 0; だとレイアウトが崩れるため微調整 */
    top: -1px;
    left: -1px;
    width: calc(100% + 2px);
    height: calc(100% + 2px);
    content: "";
    border-radius: 8px;
    background: linear-gradient(90deg, #45D1D1 0%, #ED5D5D 51.56%, #F8BF2D 100%);
```
少し無理やりな実装ですが、この方法であればinputのサイズを変えることなく
枠線の色を変更できると思います(figmaの方が元のデザインで正しいのであれば気にしなくて大丈夫です)。

<img width="1512" alt="スクリーンショット 2023-01-31 15 21 37" src="https://user-images.githubusercontent.com/103909058/215682360-64e14321-2337-42bb-9f8d-12ed5a54e540.png">

Reply:
また`--main-heading-decoration`とbackgroundの色が同じなので`--main-heading-decoration`を使用するか
名前が気になる場合は別の変数としてサークル用のCSSを宣言してもいいと思います。